v5.2.0 (#123)
LycheeOrg/LycheeOrg.github.io: docs/releases.md

## Version 5

### v5.2.0

Released on Apr 06, 2024

#### Anyone running version 5 should update without delays.

We strongly recommend anyone using Lychee version 5 to migrate as soon as possible to 5.2.
There are major bug fixes contained in this update, most notably when uploading multiple images at the same time.

#### The current LycheeOrg situation.

> We are currently 2 active members (d7415 and myself). As a result the development are significantly slowed down.

Reviewer: ```suggestion
We are currently 2 active members (d7415 and myself). As a result development has been significantly slowed down.
```

> If you like Lychee and wish to contribute, fixes bug, add new features, you will be more than welcome to join the team

Reviewer: ```suggestion
If you like Lychee and wish to contribute, fix bugs and add new features you are more than welcome to join the team
```
